ethpm/utils/contract.py

```diff
@@ -4,19 +4,25 @@
     Any,
     Dict,
     Generator,
+    List,
     Tuple,
 )
 
-from ethpm.exceptions import ValidationError
-
-from web3 import Web3
-
 from eth_utils import (
     to_bytes,
     to_dict,
-    encode_hex
+    encode_hex,
 )
 
+from solc import compile_files
+
+from web3 import Web3
+
+from ethpm.exceptions import ValidationError
+
+
+PACKAGES_DIR = 'ethpm/assets/v2-packages/'
```

Comment:
This will break in many cases.  My preferred way to do this is:

```python
import ethpm

BASE_DIR = os.path.dirname(ethpm.__file__)  # gets the root directory that the `ethpm` module is in.

V2_PACKAGES = os.path.joins(BASE_DIR, 'ethpm', 'assets', 'v2-packages')
```

Reply:
Or the fancy [pathlib](https://docs.python.org/3/library/pathlib.html), like:

```py
from pathlib import Path

BASE_DIR = Path(ethpm.__file__).parent

V2_PACKAGES = BASE_DIR / 'ethpm' / 'assets' / 'v2-packages'
```